### PR TITLE
Extend XML examples

### PR DESCRIPTION
**Pull request by EduPonz**
This PR extends XML examples to include different profile types selected by name.

Signed-off-by: EduPonz <eduardoponz@eprosima.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,15 +87,15 @@ For doing so, `rmw_fastrtps` locates profiles in the XML based on topic names ab
 
 ##### Creating publishers/subscriptions with different profiles
 
-To configure a publisher/subscription, define a `<publisher>`/`<subscriber>` profile with attribute `profile_name=topic_name`.
+To configure a publisher/subscription, define a `<publisher>`/`<subscriber>` profile with attribute `profile_name=topic_name`, where topic name is the name of the topic before mangling, i.e. the topic name used to create the publisher/subscription on `rclcpp`/`rclpy`.
```
Comment (hidmic) [inline README.md:90]: @EduPonz consider not mentioning client libraries directly to stay general (as examples at most)

**Review by hidmic — APPROVED** [review]
Overall LGTM

File: README.md (hunk 1 above)
Comment (EduPonz) [reply README.md]: Done in 4909e2b

**Review by hidmic — APPROVED** [review]
LGTM

**Pull request merged**